### ISSUE TITLE
a11y: Missing alt attribute for informative images

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12810]
a11y issue: 157

The "alt" attribute is not provided for the informative image, resulting in screen reader announcing the images as unlabelled to their users.

Occurrences to be fixed:

Cluster Tools:
Main Content
- CIS Benchmark
- Elemental
- Istio
- Longhorn and so on logos

Charts
Main Content
- CIS Benchmark
- Elemental
- Istio
- Longhorn and so on logos

Charts: Kubecost (2.4.1)
Main content
- Logo Image 

Cluster Management - Create
- Amazon EKS
- Azure AKS
- Google GKE
- Amazon EC2
- Azure and so on logo

Extensions
Main Content 
- AKS Provisioning
- EKS Provisioning
- GKE Provisioning
- Virtualization Manager

Example:

![image](https://github.com/user-attachments/assets/e1548ddc-02bd-41aa-811e-ef40efe54aa4)

**Comment by izaac** [comment]
Rancher version: `v2.11-bff2809d433c014a595d024f76fd7eabbbfa3013-head`
UI Version: `master ffe0996`
Chrome version: `133.0.6943.126 `
Firefox version: `135.0.1 `

I've validated the icons alt text is getting picked and read out loud by screen reader extensions. Although navigating with the keyboard sometimes miss to select the inner image of a section this works by clicking over the images with the mouse.

Icons alt text validated include the sections described in this issue.
